src/django/api/templates/mail/claim_facility_denial_body.html

```diff
@@ -21,16 +21,17 @@
                 Facility URL: {{ facility_url }}
             </li>
         </ul>
-        {% if denial_reason|length %}
         <p>
             Here's the reason for the denial:
         </p>
         <p>
-            {{ denial_reason }}
+            The claim information is inaccurate. If you believe this denial to be inaccurate,
```

Comment:
This change matches what was requested in the issue, but I want to confirm that there was some discussion about not having the reason entered into the approval form appear in the email. 

The denial form now incorrectly says that the content will be mailed to the claimant.

<img width="586" alt="Screen Shot 2019-06-28 at 10 10 50 AM" src="https://user-images.githubusercontent.com/17363/60359161-0ed83680-998d-11e9-9d6a-2712776086b3.png">